Makefile

```diff
@@ -16,32 +16,38 @@ TESTS := false
 # The endpoint uses HTTP connection to transmit encrypted data by default.
 # See "HTTPS Connection Support" in docs/endpoint.md for more information.
 ENFORCE_EP_HTTPS := false
-# The CFLAGS to pass during endpoint app build
-ENDPOINT_CFLAGS :=
+# The CFLAGS to pass into endpoint build process for mkapp.
+# The mkapp is a tool provides by Legato project.
+# The "-C" is the flag for mkapp to specify extra flags to be passed to the C compiler.
+LEGATO_FLAGS :=
 DEPS += $(DCURL_LIB)
 
 # Determine to enable HTTPS connection
 ifeq ($(ENFORCE_EP_HTTPS), true)
-	ENDPOINT_CFLAGS += -C -DENDPOINT_HTTPS
```

Comment:
Alternatively, we can use GNU Make macros to write a simple preprocessor which accepts typical CFLAGS and converts into mkapp-friendly `-C` list. That would be elegant.

Reply:
Reference: https://www.gnu.org/software/make/manual/html_node/Foreach-Function.html